ServerSide/yoty/WebApi/PublicDataSchemas/DTO/ProductDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) YOTY Corporation and contributors. All rights reserved.
+
+namespace YOTY.Service.WebApi.PublicDataSchemas
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class ProductDTO
+    {
+        public string Name { get; set; }
+
+        public List<Uri> Images { get; set; }
```

Comment:
Please change this to a single URI and rename to Image
If we use a list then we will need to add a new db table (we said we'll start with a single pic)